pylti/tests/test_flask.py

```diff
@@ -173,9 +183,11 @@ def test_access_to_oauth_resource(self):
         self.assertFalse(self.has_exception())
 
     def test_access_to_oauth_resource_name_passed(self):
+        # pylint: disable=invalid-name
```

Comment:
I think you are still getting these because you have allowed longer function names, but these are methods, not functions. Again, I think the entire contents of your pylintrc file should be:
```ini
[BASIC]

# Regular expression which should only match correct method names
method-rgx=[a-z_][a-z0-9_]{2,40}$
```

and possibly
```ini
[BASIC]

# Regular expression which should only match correct method names
method-rgx=[a-z_][a-z0-9_]{2,40}$

# Regular expression matching correct function names
function-rgx=[a-z_][a-z0-9_]{2,40}$
```
All of that extra stuff in that file is default, and thus not needed.